share/man/man7/release.7

```diff
@@ -283,6 +283,16 @@ This is intended for use only when
 .Fa /usr/ports
 is expected to exist by alternative means.
 .El
+.Sh REPRODUCIBLE BUILDS
+Different builders can produce bit-identical release tarballs using
+.Va PKG_TIMESTAMP
+and
+.Va REVISION :
```

Comment:
We probably want to differentiate `PKG_TIMESTAMP` from `REVISION` -- the former is an actual (literal) env variable used by Makefile*, but the latter is a placeholder in the example command below. I'm not sure off hand what the markup for that should be.

I wonder if we should have newvers.sh collect the commit time (e.g. from `git show -s --format=%ct`) and have the default SOURCE_DATE_EPOCH derived from that.